lib/auth-manager.js

```diff
@@ -78,7 +89,9 @@ AuthManager.prototype = {
     while ((plugin = this.pendingPlugins.pop()) !== undefined) {
       try {
         const authenticationHandler = yield plugin.authenticationModule(plugin,
-                this.configuration, config);
+                                                                        this.configuration,
+                                                                        config,
+                                                                        new AuthPluginContext(plugin));
```

Comment:
Formatting question: do we like this style? To me it looks like there's a huge void where the code should be, and then a bunch of code pieces hanging somewhere on the right.
```
foo(a,
<+4 spaces>b,
<+4 spaces>c);
```
or 
```
foo(
<+4 spaces>a,
<+4 spaces>b,
<+4 spaces>c);
```
look less like ASCII art to me.

Reply:
I like either, as long as observing the 120 char limit, otherwise your suggested style solves it.